### PR TITLE
Enable and disable plugins via the API

### PR DESCRIPTION
**Pull request by SchrodingersGat**
This PR does the following:

- Plugin activation is now handled directly from the settings page
- Plugin activation uses the API
- Plugins can now also be *deactivated* via the API
- Renders table of plugins via API access, rather than templated on the server side (because who wants templates?)

### CODE REVIEW

File: InvenTree/plugin/serializers.py [hunk 1]
```diff
@@ -171,6 +171,26 @@ def save(self):
 
 class PluginConfigEmptySerializer(serializers.Serializer):
     """Serializer for a PluginConfig."""
+    ...
+
+
+class PluginActivateSerializer(serializers.Serializer):
+    """Serializer for activating a plugin"""
```
Comment (matmair) [inline InvenTree/plugin/serializers.py:178]: Question: Should this maybe be renamed to reflect that it can activate/deactivate a plugin?

**Review by matmair — APPROVED** [review]
looks good in general

File: InvenTree/plugin/serializers.py (hunk 1 above)
Comment (SchrodingersGat) [reply InvenTree/plugin/serializers.py]: Done :)

**Review by matmair — APPROVED** [review]

**Comment by SchrodingersGat** [comment]
For reference, here's a screenshot of the new interface:

image

**Pull request merged**